openshift-start-configured-cluster

```diff
@@ -63,18 +63,14 @@ oc adm ca create-server-cert \
 
 openssl x509 -in $OC_CONFIG/keycloak.crt -out $OC_CONFIG/cert-only.crt
 
-X509_CERT=`cat $OC_CONFIG/cert-only.crt`
-X509_KEY=`cat $OC_CONFIG/keycloak.key`
-X509_CA=`cat $OC_CONFIG/openshift-apiserver/ca.crt`
+oc create secret tls keycloak-server-tls --cert=$OC_CONFIG/cert-only.crt --key=$OC_CONFIG/keycloak.key
+# Key is not really necessary here... although it's required by a tls secret
+oc create secret tls keycloak-client-tls --cert=$OC_CONFIG/kube-apiserver/ca.crt --key=$OC_CONFIG/kube-apiserver/ca.key
```

Comment:
What is this ???

Reply:
The line 66 contains a private key and cert for serving HTTPS in Keycloak.

The line 68 contains a CA Bundle. Your comment suggests that I should just use a secret and do not touch the `ca.key`. I'm on it!